test_cli.py

```diff
@@ -1619,6 +1618,23 @@ def test(ctx):
     # run_command([cl, 'edit', uuid], 1)
 
 
+@TestModule.register('workers')
+def test(ctx):
+    # Run workers command
+    result = run_command([cl, 'workers'])
+    # Output should contain 3 lines
```

Comment:
I don't you can assume there's exactly one worker (in any case, should document why 3).  Just assume there's at least one.

Reply:
I see. I ran test locally and it always returns me 1 worker. So I just put 3 lines there.